stage0/src/sev.rs

```diff
@@ -580,6 +589,32 @@ pub fn validate_memory(e820_table: &[BootE820Entry], encrypted: u64) {
         }
     }
 
+    // Sanity check that we have a valid legacy SMBIOS range entry in the E820
+    // table.
+    assert!(legacy_smbios_range_size == 0x1_0000);
```

Comment:
If you go with the suggestions above, this is probably not needed since the size can be included in the criteria for finding the entry.

Reply:
Done.